src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -67,7 +67,7 @@ INSERT INTO tab_species VALUES('Mouse','Mouse','Mouse',7,0,'Mouse',0);
 INSERT INTO tab_species VALUES('Rabbit','Rabbit','Rabbit',10,0,'Rabbit',0);
 INSERT INTO tab_species VALUES('Rat','Rat','Rat',6,0,'Rat',0);
 INSERT INTO tab_species VALUES('Cat','Cat','Cat',11,0,'Cat',0);
-INSERT INTO tab_species VALUES('Cattle','Cattle','Cattle',12,0,'Cattle',0);
+INSERT INTO tab_species VALUES('Cattle','Cattle (IV only)','Cattle',12,0,'Cattle',0);

```

Comment:
Changed the display name, so that it's immediately clear that we cannot use cattle with oral applications